packages/fxa-auth-server/lib/senders/emails/bin/mjml-sever.ts

```diff
@@ -0,0 +1,56 @@
+import http from 'http';
```

Comment:
Initially I tried to setup a storybook build of the templates without making use of a mjml server, but since MJML doesn't render in browser context I used Les's [starter repo](https://github.com/lmorchard/mjml-in-storybook) as a base to deploy storybook in fxa-auth-server

Reply:
Hence my confusion! 😛